docs/Other SDL Features/Using App Services/index.md

```diff
@@ -127,15 +163,37 @@ let capabilityReason = aCapability.updateReason;
 // The app service record will give you access to a service's generated id, which can be used to address the service directly (see below), it's manifest, used to see what data it supports, whether or not the service is published (if it's not, it was just removed and should not be addressed), and whether or not the service is the active service for its service type (only one service can be active for each type)
 let serviceRecord = aCapability.updatedAppServiceRecord;
 ```
+!@
 
+@![android,javaSE,javaEE]
+##### Java
+```java
+// This array contains all currently available app services on the system
+List<AppServiceCapability> appServices = servicesCapabilities.getAppServices();
+
+if (appServices!= null && appServices.size() > 0) {
+    for (AppServiceCapability anAppServiceCapability : appServices) {
+        // This will tell you why a service is in the list of updates
+        ServiceUpdateReason updateReason = anAppServiceCapability.getUpdateReason();
+
+        // The app service record will give you access to a service's generated id, which can be used to address the service directly (see below), it's manifest, used to see what data it supports, whether or not the service is published (it always will be here), and whether or not the service is the active service for its service type (only one service can be active for each type)
+        AppServiceRecord serviceRecord = anAppServiceCapability.getUpdatedAppServiceRecord();
+    }
+}
+```
+!@
+
+@![iOS]
 #### Using the System Capability Manager
 In the current release, while the system capability manager supports getting the app services capability, it does not support automatically updating itself when the app services change. It must currently be polled through the `updateCapabilityType:completionHandler:` method. Therefore, until the `SystemCapabilityManager` is updated with support for auto-updating, we currently recommend using the raw `SDLGetSystemCapability` RPC and subscribing.
+!@
```

Comment:
This iOS section no longer applies as of the current release and can be removed.

Reply:
removed